data/crypt4gh/README.md

```diff
@@ -0,0 +1,34 @@
+# Crypt4GH example file
+
+This is just a customised summary for htsget-rs. Please refer to the official [`crypt4gh-rust` documentation](https://ega-archive.github.io/crypt4gh-rust) for further information.
+
+## Keygen
+
+```sh
+cargo install crypt4gh
+crypt4gh keygen --sk keys/alice.sec --pk keys/alice.pub
+crypt4gh keygen --sk keys/bob.sec --pk keys/bob.pub
+```
+
+## Encrypt
+```
+crypt4gh encrypt --sk keys/alice.sec --recipient_pk keys/bob.pub < htsnexus_test_NA12878.bam > htsnexus_test_NA12878.bam.c4gh
+```
+
+## Decrypt
+
+```sh
+crypt4gh decryptor --range 0-65535 --sk data/crypt4gh/keys/bob.sec \
```

Comment:
@mmalenic Wouldn't it be a more interesting demo to pass an index name (i.e `chr1`) instead of manually passing a range manually? ... can this PR/htsget do curl `https://localhost/reads/bam_name/chr1` for an arbitrarily encrypted example `.bam` file? Sorry if I missed some context/explanation, this PR is pretty big :-!

Reply:
Yes, that's the idea (sorry I hadn't updated that README.md)! This only works in relation to a `UrlStorage` backend which can encrypt data for htsget-rs. So to demonstrate this with `curl`, a mock backend would need to be setup, which returns encrypted data that htsget-rs can read. This is more or less what the test in [this](https://github.com/umccr/htsget-rs/pull/223#discussion_r1452880329) comment does.